### DISCUSSION TITLE
read_def ERROR (DEFPARS-6010)

### DISCUSSION THREAD

**Discussion by qujijie** [The-OpenROAD-Project/OpenROAD#3401] [Q&A] [answered]
when read_def I have this error. I don't know how to fix it.
[WARNING ODB-0126] error: TECHNOLOGY is unsupported
ERROR (DEFPARS-6010): An error has been reported in callback. Error in file /home/qjj/CodeFile/OpenROAD/qjj/test/threestage/ThreeStage_4cbe5670900884f3.def at line 5, on token ;.
Update the def file before parsing the file again.

[ThreeStage_def.zip](https://github.com/The-OpenROAD-Project/OpenROAD/files/11600994/ThreeStage_def.zip)

**Comment by maliberty** [comment, marked as answer]
This should have been opened as an issue with the full set of data.  I believe you are on an old version of OR as the current message is a warning "TECHNOLOGY is ignored".  If you see the same with the head of master please open an issue.
